### PR TITLE
Replace Sami by Doctum

### PR DESCRIPTION
**Pull request by williamdes**
Fixes: https://github.com/easyrdf/easyrdf/issues/322

### CODE REVIEW

**Review by k00ni — APPROVED** [review]
Hey @williamdes, thank you for the PR! Because I have no further knowledge in Sami or Doctum, I would like to hear @njh's opinion on this.

**Review by k00ni — COMMENTED** [review]
*Misclicked on approve*

**Review by k00ni — CHANGES_REQUESTED** [review]
Sorry for the spam, had to revert the approval because of a misclick.

* [ ] feedback of @njh

**Comment by njh** [comment]
This is great, thanks @williamdes !

I really like the style of Sami more than the others, so great to be able to continue using it.

File: CHANGELOG.md [hunk 1]
```diff
@@ -24,6 +24,7 @@ Enhancements
 * Updated to PHP CodeSniffer v3
 * Updated to Sami v4 for API documentation
 * Updated to PHPUnit v7
+* Replace abandonned Sami by Doctum
```
Comment (njh) [inline CHANGELOG.md:27]: Version 1.0.0 has already been released.

We need to add a new section to the ChangeLog for the changes since 1.0.0

Comment (k00ni) [reply CHANGELOG.md:27]: @williamdes can you add a `1.1.0` section with your addition?

@njh after we merged this, we could add further content to this section and release it. What do you think?

Comment (williamdes) [reply CHANGELOG.md]: Done :)

**Comment by williamdes** [comment]
> This is great, thanks @williamdes !
> 
> I really like the style of Sami more than the others, so great to be able to continue using it.

Thank you :)
I fixed my commit and force pushed.

**Review by njh — APPROVED** [review]

**Comment by williamdes** [comment]
Rebased with origin/master

**Review by njh — APPROVED** [review]

**Review by k00ni — APPROVED** [review]

**Comment by williamdes** [comment]
Let me know when it is deployed :)

**Comment by k00ni** [comment]
> Let me know when it is deployed :)

Do you mean when we released a new version?

**Comment by williamdes** [comment]
> > Let me know when it is deployed :)
> 
> Do you mean when we released a new version?

When https://www.easyrdf.org/docs/api/classes.html is updated

**Comment by k00ni** [comment]
@njh Can you handle that?

**Comment by njh** [comment]
Perhaps not ideal, but the website will be updated when we do the next release of EasyRdf.

We should attach some tickets to the 1.1 milestone and make a release when they are all done...

**Comment by njh** [comment]
Doctum generated API docs are now published to the website: https://www.easyrdf.org/docs/api/classes.html

**Pull request merged**